contrib/win32/win32compat/termio.c

```diff
@@ -247,6 +245,19 @@ syncio_initiate_write(struct w32_io* pio, DWORD num_bytes)
 	return 0;
 }
 
+static BOOL
```

Comment:
declare it in misc_internals.h and define in misc.c so that we can reuse..

Reply:
change to use  IsWindows8OrGreater()